packages/react-dom/src/events/DOMEventProperties.js

```diff
@@ -73,27 +60,7 @@ const discreteEventPairsForSimpleEventPlugin = [
   ('touchend': DOMEventName), 'touchEnd',
   ('touchstart': DOMEventName), 'touchStart',
   ('volumechange': DOMEventName), 'volumeChange',
-];
-
-const otherDiscreteEvents: Array<DOMEventName> = [
```

Comment:
These go away from this file because they're not SimpleEventPlugin events. They were only listed here to give them a discrete priority, which we do in the new switch.